### ISSUE TITLE
Shakedex Auction documentation

### ISSUE THREAD

**Issue by H7-25** [kyokan/bob-wallet#361]
Nowhere is it specified what happens when an auction in the shakedex exchange ends without any offer and how to get the domain back in your wallet

**Comment by rithvikvibhu** [comment]
Yeah we need docs on the Exchange tab and Shakedex in general. Maybe a wiki page on the whole process. Let's leave this issue open until that's written.

---

>Nowhere is it specified what happens when an auction in the shakedex exchange ends without any offer and how to get the domain back in your wallet

Shakedex auctions don't really end, the lowest/ending price value stays until someone buys it or is cancelled. And yeah, Bob will show a "cancel" that will transfer the name back to your wallet.
